osu.Game.Rulesets.Osu/Mods/OsuModFlashlight.cs

```diff
@@ -48,7 +52,16 @@ public void OnSliderTrackingChange(ValueChangedEvent<bool> e)
 
             protected override bool OnMouseMove(MouseMoveEvent e)
             {
-                FlashlightPosition = e.MousePosition;
+                var position = FlashlightPosition;
+                var destination = e.MousePosition;
+
+                double frameTime = Clock.CurrentTime - lastPositionUpdate;
+                double interp = Interpolation.ApplyEasing(Easing.Out, MathHelper.Clamp(frameTime / follow_delay, 0, 1));
+                FlashlightPosition = new Vector2(
+                    (float)Interpolation.Lerp(position.X, destination.X, interp),
+                    (float)Interpolation.Lerp(position.Y, destination.Y, interp)
+                );
+                lastPositionUpdate = Clock.CurrentTime;
```

Comment:
I believe this is all equivalent to:

```
FlashlightPosition = Interpolation.ValueAt(MathHelper.Clamp(Clock.ElapsedFrameTime, 0, follow_delay), position, destination, 0, follow_delay, Easing.Out)
```

Reply:
Not sure how I missed ValueAt, changed it now.